### DISCUSSION TITLE
Choose the responsive calculator code

### DISCUSSION THREAD

**Discussion by DhanushNehru** [DhanushNehru/calculator#22] [Polls]
I could see two responsive calculator code here 

[Calculator 1](https://deploy-preview-19--calculator-dhanushnehru.netlify.app/)  by @sauravsilawat

[Calculator 2](https://deploy-preview-18--calculator-dhanushnehru.netlify.app/) by @subhranshuchoudhury

Which one would you pick ?

**Comment by SmitaChoudhury** [comment]
![Screenshot_2022-10-04-23-16-49-478_com android chrome](https://user-images.githubusercontent.com/93305945/193889894-e69829a3-5a5d-40a4-8365-e64e896a9288.jpg)

@sauravsilawat

**Comment by SmitaChoudhury** [comment]
![Screenshot_2022-10-04-23-17-31-126_com android chrome](https://user-images.githubusercontent.com/93305945/193889997-8f5fbbb6-9112-4954-9a4a-11c969b915d0.jpg)
@subhranshu Choudhury
